proto/shentu/oracle/v1alpha1/query.proto

```diff
@@ -26,16 +26,16 @@ service Query {
         option (google.api.http).get = "/shentu/oracle/v1alpha1/contract/{contract}/function/{function}/task";
     }
 
-    rpc TxTask(QueryTxTaskRequest) returns (QueryTxTaskResponse) {
-        option (google.api.http).get = "/shentu/oracle/v1alpha1/txhash/{tx_hash}/txtask";
+    rpc AtxTask(QueryAtxTaskRequest) returns (QueryAtxTaskResponse) {
```

Comment:
Shall we have this renamed?
I guess it could be `TxTask` because it handles transactions instead of smart contracts which are taken care of by `Task`.
I guess we should do the Atx renaming only for the original tx_byte and tx_hash, because they are actually not the transaction itself, but a wrapping structure carrying application transaction.

Reply:
We also need to differentiate application chain transaction and shentu transaction. 
tx indicates shentu transaction.

Reply:
I think we don't have to think about Shentu transaction when we talk about Oracle module. 
In my personal opinion, there is only one kind of transaction, i.e. the transaction that is going to be evaluated.